README.md

```diff
@@ -1,47 +1,41 @@
+# Smppex
+
 [![Build Status](https://travis-ci.org/funbox/smppex.svg?branch=master)](https://travis-ci.org/funbox/smppex)
 [![Documentation](https://img.shields.io/badge/docs-hexpm-blue.svg)](http://hexdocs.pm/smppex)
 [![Version](https://img.shields.io/hexpm/v/smppex.svg)](https://hex.pm/packages/smppex)
 [![Coverage Status](https://coveralls.io/repos/github/funbox/smppex/badge.svg?branch=master&1504538909)](https://coveralls.io/github/funbox/smppex?branch=master)
 [![Inline docs](http://inch-ci.org/github/funbox/smppex.svg?branch=master)](http://inch-ci.org/github/funbox/smppex)
 
-<a href="https://funbox.ru">
-  <img src="http://funbox.ru/badges/sponsored_by_funbox_compact.svg" alt="Sponsored by FunBox" width=250 />
-</a>
-
-# Smppex
-
-SMPP 3.4 protocol and framework implementation in [Elixir](http://elixir-lang.org)
+SMPP 3.4 protocol and framework implementation in [Elixir](http://elixir-lang.org).
 
-See [Examples](https://hexdocs.pm/smppex/SMPPEX.html)
+See [Examples](https://hexdocs.pm/smppex/SMPPEX.html) for details.
 
 ## Documentation
 
-API documentation is available at http://hexdocs.pm/smppex
+API documentation is available at **[hexdocs.pm/smppex](http://hexdocs.pm/smppex)**.
 
 ## Live Demo
 
-There is a simple online demonstrational MC (SMPP server) at http://smppex.rubybox.ru
+There is a simple online demo MC (SMPP server) at **[smppex.rubybox.ru](http://smppex.rubybox.ru)**.
 
 ## Installation
 
-The package can be installed as:
-
-  1. Add `smppex` to your list of dependencies in `mix.exs`:
+To install and use the package:
 
-  ```elixir
-  def deps do
-    [{:smppex, "~> 2.0"}]
-  end
-  ```
+1. Add `smppex` to your list of dependencies in `mix.exs`:
 
-  2. Ensure `smppex` is started before your application:
+   ```elixir
+   def deps do
+     [{:smppex, "~> 2.0"}]
+   end
+   ```
 
-  ```elixir
-  def application do
-    [applications: [:smppex]]
-  end
-  ```
+2. Ensure `smppex` is started before your application:
 
-## LICENSE
+   ```elixir
+   def application do
+     [applications: [:smppex]]
+   end
+   ```
 
-This software is licensed under [MIT License](LICENSE).
```

Comment:
What is reason to remove mention of the license terms from the readme?

Reply:
There's license file in the repo: https://github.com/funbox/smppex/blob/master/LICENSE.

I've removed the line just to make the README clear. I may put it back if you find it useful there.

Reply:
When browsing repositories, I find it convenient to learn about license in the same place that describes purpose of the library.
But that's only my preference, others may have different opinion.

Reply:
Well, if someone finds it useful, why not to put it back. I've reverted it.

Reply:
Thanks. Have a nice day!
Peter